ui/src/app/pages/login/account-creation/account-creation-institution.tsx

```diff
@@ -235,12 +314,21 @@ export class AccountCreationInstitution extends React.Component<Props, State> {
                                     This will be the primary email contact for your new account.
                                   </div>
                                 </div>}
-                                invalid={this.state.emailFailedValidation}
-                                onBlur={() => this.validateContactEmailInline()}
-                                onChange={email => this.updateContactEmail(email)}/>
-            {this.state.emailFailedValidation &&
-              <ErrorDiv data-test-id='invalid-email-error'>
-                Error: email address is invalid
+                                invalid={!this.isEmailValid()}
+                                onBlur={() => this.onEmailBlur()}
```

Comment:
Is this event guaranteed to happen before the submit button is clicked?  Consider whether a user might choose to enter their Role before their Email

Reply:
That situation is caught correctly by the form-level validation: if an institution is selected, the form won't allow progress unless there is a non-null `checkEmailResponse` object in the component state.